server/store.go

```diff
@@ -30,21 +30,16 @@ func hashkey(prefix, key string) string {
 	return fmt.Sprintf("%s%x", prefix, h.Sum(nil))
 }
 
-func (store store) get(key string, v interface{}) (returnErr error) {
-	defer func() {
-		if returnErr == nil {
-			return
-		}
-		returnErr = errors.WithMessage(returnErr, "failed to get from store")
-	}()
+var ErrUserNotFound = errors.New("user not found")
 
+func (store store) get(key string, v interface{}) error {
 	data, appErr := store.plugin.API.KVGet(key)
 	if appErr != nil {
-		return appErr
+		return errors.New(appErr.Error())
```

Comment:
nit: why errors.New? for the callstack?

Reply:
@levb because it's returning an error, not an appErr

Reply:
Okay, fixed, thanks!